### PR TITLE
fix(webview): prevent keyboard from obscuring inputs in webview

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Previously inputs in the webview could become obscured by the keyboard display. For inputs above where the keyboard would display such as Staked Celo there is no change.

#### Impact Market Android Before
https://github.com/valora-inc/wallet/assets/26950305/b3b84c67-1270-4851-92f3-1230622aeedf

#### Impact Market Android After
https://github.com/valora-inc/wallet/assets/26950305/10c7fa02-c6b5-4a97-b257-009cfde45dc8

#### Wallet DM Android Before
https://github.com/valora-inc/wallet/assets/26950305/dfc326a8-837b-42f7-abca-8547bd85e11c

#### Wallet DM Android After
https://github.com/valora-inc/wallet/assets/26950305/84956572-b90f-4362-b3a5-559bc90f707a

### Test plan

- Tested locally on iOS
- Tested locally on Android

### Related issues

N/A

### Backwards compatibility

Yes

### CODE REVIEW

**Review by kathaypacific — APPROVED** [review]
nice!! LGTM :)

File: src/webview/WebViewScreen.tsx [hunk 1]
```diff
@@ -195,21 +204,27 @@ function WebViewScreen({ route, navigation }: Props) {
...
-          setCanGoBack(navState.canGoBack)
-          setCanGoForward(navState.canGoForward)
-          handleSetNavigationTitle(navState.url, navState.title, navState.loading)
-        }}
-        mediaPlaybackRequiresUserAction={mediaPlaybackRequiresUserAction}
-        testID={activeDapp ? `WebViewScreen/${activeDapp.name}` : 'RNWebView'}
-      />
+      <KeyboardAvoidingView
+        style={styles.keyboardView}
+        behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
```
Comment (jeanregisser) [inline src/webview/WebViewScreen.tsx:209]: Is this really needed on iOS? I just tried quickly on iOS and inputs are already avoiding the keyboard.

Comment (MuckT) [reply src/webview/WebViewScreen.tsx:209]: Do you mean the behavior or the whole `KeyboardAvoidingView`?

Comment (MuckT) [reply src/webview/WebViewScreen.tsx:209]: I don't think it's needed on iOS, but I also don't see any detriment in adding it. I tested quickly on iOS by using Withdraw > Spend > Bidali > Amazon Gift Card > Custom Amount.

Comment (jeanregisser) [reply src/webview/WebViewScreen.tsx:209]: Sounds good, thanks for the details 👍

**Pull request merged**